lib/active_proxy/proxy.ex

```diff
@@ -22,12 +22,16 @@ defmodule ActiveProxy.Proxy do
   end
 
   defp start_serve_process(socket) do
+    timeout = Application.get_env(:active_proxy, :timeout)
+    application_address = Application.get_env(:active_proxy, :node1_address)
```

Comment:
Can you look into if we can have these as constants on the module? and for the module to eval them on startup?

Otherwise, an init on the module should accept a config object and they should be set as constants there